index.html

```diff
@@ -0,0 +1,245 @@
+          </div>
+        </div>
+
+        <div class="navbar navbar-expand-lg bg-body-tertiary navbar-menu">
+          <div class="container-fluid">
+            <div class="d-flex nav-bar-about">
+              <a class="navbar-brand logo" href="index.html">ASADinc.</a>
+              <ul class="navbar-nav nav-menu">
+                <li class="nav-item nav-item-home">
+                  <a class="nav-link link-menu about-link" aria-current="page"
+                    href="./about.html">About</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link link-menu" href="#program">Program</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link link-menu" href="#join">Join</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link link-menu" href="#speakers">Speakers</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link link-menu" href="#">News</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link link-menu link-campaign" href="#">Campaign</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="intro-main">
+        <div class="intro">
+          <h2 class="greet">"Hello! Truth Lovers"</h2>
+          <h1 class="title">CREATIONISM vs EVOLUTION<br>GLOBAL SUMMIT<br>2023</h1>
+          <p class="description">
+            A high-impact study focused on discovering the truth and
+            transmitting it to the new generations,
+            scientists from different specialties meet together with people from
+            more than 50 countries,
+            is being held in August, in Singapore.
+          </p>
+          <p class="date">2023.04.20(THU) ~ 21(FRI)</p>
+          <p class="place">@Raffles City Convention Centre, Singapore.</p>
+        </div>
+      </div>
+
+      <section class="main-program" id="program">
+        <div class="main-title">
+          <h3 class="main-title-h3">Main Program</h3>
+        </div>
+        <div class="main-bar"></div>
+        <div class="main-activities">
+          <table class="table table-borderless t-main">
+            <tbody>
+              <tr class="main-row">
+                <td class="t-img"><img class="main-img" src="Images/main_01.png"
+                    alt="Speaker Icon"></td>
+                <td class="t-title">Lecture</td>
+                <td class="t-description">Listen to the speakers from various
+                  countries about
+                  they cientific studies related to evolution.</td>
+              </tr>
+            </tbody>
+          </table>
```

Comment:
- Kindly, these main program lists are articles so make them inside `article` for semantic reasons 👍